### DISCUSSION TITLE
Welcome to resiprocate Discussions!

### DISCUSSION THREAD

**Discussion by sgodin** [resiprocate/resiprocate#166] [General]
## 👋 Welcome!
We are have fully transitioned away from the legacy reSIProcate mailman mailing lists for community discussions (https://list.resiprocate.org/mailman/listinfo), and are now using the GitHub Discussions for all future discussions.

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sgodin** [comment]
We have officially transitioned over to GitHub Discussions, please don't post to older mailman mailing lists any longer.  :)

**Reply by mutable-dan** [reply]
Glad I saw this, I was reluctantly considering signing up on the mailing list. I just forked the project.  It looks llike the project is still active? I am considering getting to know the project and hopefully contributing.

**Reply by sgodin** [reply]
Hi @mutable-dan - Welcome to resip!   Yes this is an active project.  :)  How do you plan to use reSIProcate?

**Reply by mutable-dan** [reply]
Possible use to handle redirect requests.  But mostly just to keep my c/c++ skills moving

**Comment by AttilaSiposAtNetcall** [comment]
Hi,

can you please update the text at the top of this page.  I almost sent a question to the mailing list!

this text:
"We are currently using reSIProcate mailman mailing lists for community discussions, see: https://list.resiprocate.org/mailman/listinfo

We may move to this new GitHub Discussions feature in the near future though. For now consider this a trial."

**Reply by sgodin** [reply]
You are not the first to be caught by this, my apologies.  Great idea - I've updated the text.  Thanks!

**Reply by AttilaSiposAtNetcall** [reply]
thanks.  I got this response as email too (which is fine)  

I guess also you want to remove the "Reply to this email directly" from the email footer?  
(but I'm not sure)  
<img width="718" height="132" alt="image" src="https://github.com/user-attachments/assets/925e94fe-2c38-4b75-9b66-2cfd7956440d" />
